### PR TITLE
Rename ZosPackageFile, ZosNetworkFile, ZosVersion and zosversion

### PR DESCRIPTION
**Pull request by ylv-io**
How funny it is that PR to rename ZOS is PR number 1000. I'd say it is a good luck.

### CODE REVIEW

File: examples/cli-create2/index.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ async function useSignedCreate2(initValue, networkConfig) {
   const initData = Sample.methods.initialize(initValue).encodeABI();
 
   // We will use the project's proxy admin as upgradeability admin of this instance
-  const networkFile = new files.ZosPackageFile().networkFile(networkConfig.network);
+  const networkFile = new files.ProjectFile().networkFile(networkConfig.network);
```
Comment (jcarpanelli) [inline examples/cli-create2/index.js]: the noun `project` is more accurate than `Package`. Loved it!

File: packages/cli/src/models/files/ProjectFile.ts [hunk 2]
```diff
@@ -58,7 +58,7 @@ export default class ZosPackageFile {
       }.`;
       throw e;
     }
-    checkVersion(this.data.zosversion, this.fileName);
+    checkVersion(this.data.manifestVersion, this.fileName);
```
Comment (jcarpanelli) [inline packages/cli/src/models/files/ProjectFile.ts]: Are we still going to support reading the `zosversion` attribute after these changes for users that have already started their projects with `zos` versions below `2.4.0`?

Comment (ylv-io) [reply packages/cli/src/models/files/ProjectFile.ts]: I don't think we should support reading old `zosversion`. I belive we should migrate them to new `manifestVersion`. @spalladino What do you think?

**Comment by ylv-io** [comment]
This is pretty big PR and I don't want to add more stuff on top of it. Shall we merge it into a master even it doesn't have migration strategy for old `zosversion`? @jcarpanelli @spalladino ?

File: packages/cli/src/models/files/ProjectFile.ts (hunk 2 above)
Comment (spalladino) [reply packages/cli/src/models/files/ProjectFile.ts]: Hmm I guess we should support `zosversion` for `zos.json` files, and `manifestVersion` for `.openzeppelin/project.json` files.

**Comment by spalladino** [comment]
> Shall we merge it into the master even it doesn't have migration strategy for old zosversion? @jcarpanelli @spalladino ?

I'm fine with that @ylv-io! Make sure you keep in mind that the migration strategy is pending. We can handle it together with #931.

**Pull request merged**